src/main/java/org/acme/persistence/RepoEntityRepository.java

```diff
@@ -0,0 +1,19 @@
+package org.acme.persistence;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepository;
+import io.quarkus.panache.common.Parameters;
+import org.acme.model.Repository;
+import org.acme.model.RepositoryType;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.List;
+
+@ApplicationScoped
+public class RepoEntityRepository implements PanacheRepository<Repository> {
+
+    public List<Repository> findRepositoryByRepositoryType(RepositoryType type){
```

Comment:
The original code specifies an order for this query. 

https://github.com/DependencyTrack/dependency-track/blob/d0b179d637624250688c526bb9c740e1e1b6b4ec/src/main/java/org/dependencytrack/tasks/repositories/RepositoryMetaAnalyzerTask.java#L74